pytree/views.py

```diff
@@ -2,16 +2,17 @@
 from pytree import app
 from flask import request, render_template
 from flask import jsonify
-from os.path import abspath, dirname
+from os import path
 import subprocess
 import yaml
 from yaml import FullLoader
 from flask_cors import cross_origin
 
-yaml_config_file = (dirname(abspath(__file__)) + ".yml")
+yaml_config_file = (path.dirname(path.abspath(__file__)) + ".yml")
 with open(yaml_config_file, 'r') as f:
     pytree_config = yaml.load(f, Loader=FullLoader)
 
+import logging
```

Comment:
👍🏼

Reply:
Actually, this is a left-over. Logging is deactivated in the current state and I won't have time to do something about it right now...